extension/vendor/mooltipass/device.js

```diff
@@ -273,8 +273,11 @@ mooltipass.device.onTabClosed = function(tabId, removeInfo)
         }
         else
         {
-            /* Send a cancelling request if it is the tab from which we're waiting an answer */
-            chrome.runtime.sendMessage(mooltipass.device._app.id, {'cancelGetInputs' : {'reqid': mooltipass.device.retrieveCredentialsQueue[0].reqid, 'domain': mooltipass.device.retrieveCredentialsQueue[0].domain, 'subdomain': mooltipass.device.retrieveCredentialsQueue[0].subdomain}});             
+            // Emulation part
+            if ( !mooltipass.emulator.active ) {
+                /* Send a cancelling request if it is the tab from which we're waiting an answer */
+                chrome.runtime.sendMessage(mooltipass.device._app.id, {'cancelGetInputs' : {'reqid': mooltipass.device.retrieveCredentialsQueue[0].reqid, 'domain': mooltipass.device.retrieveCredentialsQueue[0].domain, 'subdomain': mooltipass.device.retrieveCredentialsQueue[0].subdomain}});             
+            }
```

Comment:
Not necessary, but as we're not using the queue, I wanted to prevent weirdness in the future.